### ISSUE TITLE
[BUG] [Formatter] comments containing html are not indented correctly

### ISSUE THREAD

**Issue by christopherpickering** [Riverside-Healthcare/djLint#247]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## Issue

```html
<!DOCTYPE html>
<html>
  <body>
<!-- Do not display this at the moment
<img border="0" src="pic_trulli.jpg" alt="Trulli">
-->
  <!-- Do not display this at the moment
  <img border="0" src="pic_trulli.jpg" alt="Trulli">
  -->
    <!-- Do not display this at the moment
    <img border="0" src="pic_trulli.jpg" alt="Trulli">
    -->
  </body>
</html>
```

should be formatted as

```html
<!DOCTYPE html>
<html>
    <body>
        <!-- Do not display this at the moment
        <img border="0" src="pic_trulli.jpg" alt="Trulli">
        -->
        <!-- Do not display this at the moment
        <img border="0" src="pic_trulli.jpg" alt="Trulli">
        -->
        <!-- Do not display this at the moment
        <img border="0" src="pic_trulli.jpg" alt="Trulli">
        -->
  </body>
</html>
```

but currently the contents of the  are not indented.

**Comment by christopherpickering** [comment]
contents of comments will not be formatted. changing the indentation will lead to a recurring indent.
